src/pubkey.h

```diff
@@ -205,4 +210,12 @@ struct CExtPubKey {
     bool Derive(CExtPubKey& out, unsigned int nChild) const;
 };
 
+/** Users of this module must hold an ECCVerifyHandle. */
```

Comment:
Users have to hold a ECCVerifyHandle. Maybe we should make them pass it in to make sure of that? :-)